mission.py

```diff
@@ -24,7 +17,8 @@ def __init__(self, args):
                 self.agent = default_agent()
             else:
                 agent_name = "agent{}".format(args.agent[0])
-                self.agent = eval(agent_name + "()")
+                agent_module = import_module(f".{agent_name}", "agents")
```

Comment:
I can revert this change if you prefer the previous way. This dynamically imports an agent based on the argument instead of importing all of them at once, which is helpful because different groups won't need to install each others' dependencies, and any side effects (e.g. printing at the top level of a module) won't be run.

Reply:
e.g. Group 1 currently has a `print()` call at the top level, which causes it to print any time `main.py` is run, regardless of which agent is selected. This makes that not happen since agent1 won't be imported unless you're running `main.py -a 1`.

Reply:
I see. Good idea.